test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,66 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+const md5 = require('md5');
+
+const path = 'https://api.github.com/users/aperdomob';
+const path2 = 'https://api.github.com/repos/aperdomob';
```

Comment:
This url should not be needed. Only the first one, and the other should by obtained by hypermedia

Reply:
solved